### PR TITLE
TGX/Contest-2022 #154

### PR DESCRIPTION
**Pull request by iRYO400**
Less than a half finished:
- Small(I guess) reactions in chat
- Reaction selection with animation
- and some settings

lost a lot of time at implementing big reactions in the chat, left them in the shelve

### CODE REVIEW

File: app/src/main/java/org/thunderdog/challegram/navigation/ViewController.java [hunk 1]
```diff
@@ -2248,7 +2250,8 @@ public static class Options {
     public final CharSequence info;
     public final OptionItem[] items;
 
-    public Options (CharSequence info, OptionItem[] items) {
+
+    public Options(CharSequence info, OptionItem[] items) {
```
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/navigation/ViewController.java:2254]: Thank you for the submission. Just leaving this comment to make sure you do not miss these links:

https://t.me/tgx_dev/1568
https://t.me/tgx_dev/1561

By quick jumping through the PR, I've noticed that there are several changes that could be excluded without affecting the result, like this one.

Note that they are insignificant and can be addressed after the deadline — meanwhile you may submit any commits up until the deadline if you have something more important in mind.

Comment (iRYO400) [reply app/src/main/java/org/thunderdog/challegram/navigation/ViewController.java]: Resolved, thanks!

**Comment by vkryl** [comment]
It seems that you forgot to submit the link to the @ContestBot. Could you please specify your telegram username here?

**Comment by iRYO400** [comment]
> It seems that you forgot to submit the link to the [@ContestBot](https://t.me/ContestBot). Could you please specify your telegram username here?

Hi, yes. I forgot this:
http://t.me/iRYO400

**Comment by vkryl** [comment]
Thank you for participating. Check out the results for the first round: **[🏆 Telegram X Android Competition, Round 1 Results](https://t.me/contest/297)**. Closing this pull request just to keep the pull request list clean.

**Review by Mbraj123 — COMMENTED** [review]
Hi

**Comment by Vladimirovdent** [comment]
> Hi

Hi

**Comment by Vladimirovdent** [comment]
9h3K9-qhVa6-DTn4w-TXW1Q

**Pull request closed without merging**